packages/docusaurus-plugin-content-docs/src/client/index.ts

```diff
@@ -143,9 +143,16 @@ export function useActiveVersion(
 export function useActiveDocContext(
   pluginId: string | undefined,
 ): ActiveDocContext {
-  const data = useDocsData(pluginId);
-  const {pathname} = useLocation();
-  return getActiveDocContext(data, pathname);
+  try {
+    const data = useDocsData(pluginId);
+    const {pathname} = useLocation();
+    return getActiveDocContext(data, pathname);
+  } catch (error) {
+    throw new Error(
+      'It seems that you are using a doc plugin feature while the doc plugin seems to be disabled in `docusaurus.config.js`.',
+      error as Error,
```

Comment:
```suggestion
      {cause: error as Error},
```

Error cause has to be under a cause attribute, see https://github.com/tc39/proposal-error-cause